### PR TITLE
update architecture committee meeting address for 2022

### PR DESCRIPTION
**Pull request by liubin**
Update the meeting URL to 2022

### CODE REVIEW

**Comment by liubin** [comment]
PTAL @jodh-intel

**Review by fidencio — APPROVED** [review]
lgtm, thanks @liubin!

File: src/pages/community/index.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Twitter: [@katacontainers](https://twitter.com/katacontainers)
 ## Meetings
 
 **Architecture Committee**\
-The Architecture Committee works on the technical direction of the project. Meetings are on Tuesdays at 1500 UTC. You can [add this meeting](https://zoom.us/meeting/tJUpc-2opjkjGNCIo1YFpKlcoLn909gWv_iH/ics?icsToken=98tyKuCpqj8uGtyXuRuDRowcBo-gLPTxiHpEjY1-tRC0OgZDSQ_1Oc5Ma-ImF-_G) to your calendar and find meeting agendas & call info [here](https://etherpad.opendev.org/p/Kata_Containers_2021_Architecture_Committee_Mtgs). 
+The Architecture Committee works on the technical direction of the project. Meetings are on Tuesdays at 1500 UTC. You can [add this meeting](https://zoom.us/meeting/tJUpc-2opjkjGNCIo1YFpKlcoLn909gWv_iH/ics?icsToken=98tyKuCpqj8uGtyXuRuDRowcBo-gLPTxiHpEjY1-tRC0OgZDSQ_1Oc5Ma-ImF-_G) to your calendar and find meeting agendas & call info [here](https://etherpad.opendev.org/p/Kata_Containers_2022_Architecture_Committee_Mtgs).
```
Comment (jodh-intel) [inline src/pages/community/index.md:41]: How about we have a etherpad with this URL and we embed that link into this file:

https://etherpad.opendev.org/p/Kata_Containers_Architecture_Committee_Mtgs

We could then have links in that file to each years actual etherpad.

**Review by jodh-intel — COMMENTED** [review]
Thanks @liubin - a comment...

File: src/pages/community/index.md (hunk 1 above)
Comment (liubin) [reply src/pages/community/index.md:41]: Another choice is just to use the URL without the year for the current year, at the end of this year/start of next year rename it to that year, and create a new blank page for the new year.

When the year changes, only renaming the etherpad is required.

Comment (jodh-intel) [reply src/pages/community/index.md:41]: Ack. Either option wfm ;)

Comment (liubin) [reply src/pages/community/index.md]: @jodh-intel  Updaed.

**Review by SunnyCai1 — APPROVED** [review]

**Review by jodh-intel — APPROVED** [review]
Thanks @liubin!

lgtm - I've also added some details to https://etherpad.opendev.org/p/Kata_Containers_Architecture_Committee_Mtgs (feel free to tweak of course ;)

**Pull request merged**